h2o-core/src/main/java/water/rapids/ast/prims/models/PermutationFeatureImportanceAstHelper.java

```diff
@@ -0,0 +1,33 @@
+package water.rapids.ast.prims.models;
+
+import water.fvec.Vec;
+import water.util.TwoDimTable;
+
+public class PermutationFeatureImportanceAstHelper {
```

Comment:
Dont you Ast in the name if it is not Ast class, please. 

```suggestion
public class PermutationFeatureImportanceHelper {
```